### PR TITLE
Add documentation for reload_searchIanalyzer api

### LINKED ISSUE #1269: [DOC] API - add documentation for reload_search_analyzer
**What do you want to do?**
 
- [ ] Request a change to existing documentation
- [X] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 
API - add documentation for reload_search_analyzer

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.

### PR DESCRIPTION
**Pull request by ariamarble**
### Description

Adds documentation for reload_searchIanalyzer api

### Issues Resolved

#1269 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by ariamarble** [comment]
tech review complete and tested

File: _opensearch/rest-api/reload-search-analyzer.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: Reload search analyzer
+parent: REST API reference
+nav_order: 83
+---
+
+# Reload search analyzer
+
+The reload search analyzer API operation will pick up any changes to [synonym](https://opensearch.org/docs/latest/opensearch/ux/) files for any [search analyzers](https://opensearch.org/docs/latest/im-plugin/refresh-analyzer/index/) configured. The reload search analyzer request will need to be run on all nodes. Additionally, the synonym token filter must be set to `true`.
```
Comment (Naarcha-AWS) [inline _opensearch/rest-api/reload-search-analyzer.md:10]:
```suggestion
The reload search analyzer API operation picks up any changes to [synonym](https://opensearch.org/docs/latest/opensearch/ux/) files for any configured [search analyzers](https://opensearch.org/docs/latest/im-plugin/refresh-analyzer/index/). The reload search analyzer request needs to be run on all nodes. Additionally, the synonym token filter must be set to `true`.
```

**Review by Naarcha-AWS — APPROVED** [review]
Approved, though I would remove any future tense from the description.

File: _opensearch/rest-api/reload-search-analyzer.md (hunk 1 above)
Comment (ariamarble) [reply _opensearch/rest-api/reload-search-analyzer.md:10]: will do :)

File: _opensearch/rest-api/reload-search-analyzer.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+The reload search analyzer API operation picks up any changes to [synonym](https://opensearch.org/docs/latest/opensearch/ux/) files for any configured [search analyzers](https://opensearch.org/docs/latest/im-plugin/refresh-analyzer/index/). The reload search analyzer request needs to be run on all nodes. Additionally, the synonym token filter must be set to `true`.
+
+## Path and HTTP methods
+
+```
+POST /<index>/_reload_search_analyzers
+GET /<index>/_reload_search_analyzers
+```
+
+## Request body parameters
```
Comment (kolchfa-aws) [inline _opensearch/rest-api/reload-search-analyzer.md:19]: Could we name this section "request body fields" to match our API style guide?

File: _opensearch/rest-api/reload-search-analyzer.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Field Type | Data Type | Description
+:--- | :--- | :---
+[`allow_no_indices`](#allow_no_indices) | boolean | When set to `false`, an error is returned for indicies that are closed or missing that match any wildcard expression. Default is set to `true`.
+[`expand_wildcards`](#expand_wildcards) | string | Allows you to set the wildcards that can be matched to a type of index. Available options are `open`, `closed`, `all`, `none`, and `hidden`. Default is set to `open`.
+[`ignore_unavailable`](#ignore_unavailable) | boolean | If an index is closed or missing, an error is returned when ignore_unavailable is set to `false`. Default is set to `false`.
+
+## Examples
+
+Request:
```
Comment (kolchfa-aws) [inline _opensearch/rest-api/reload-search-analyzer.md:31]: Could we add a sentence to separate the headings "examples" and "request". Also, could we make the heading "sample request" and make it a h4 (####). Same comment for the response.

**Review by kolchfa-aws — APPROVED** [review]
LGTM except minor formatting comments.

File: _opensearch/rest-api/reload-search-analyzer.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+Field Type | Data Type | Description
+:--- | :--- | :---
+[`allow_no_indices`](#allow_no_indices) | boolean | When set to `false`, an error is returned for indicies that are closed or missing that match any wildcard expression. Default is set to `true`.
+[`expand_wildcards`](#expand_wildcards) | string | Allows you to set the wildcards that can be matched to a type of index. Available options are `open`, `closed`, `all`, `none`, and `hidden`. Default is set to `open`.
+[`ignore_unavailable`](#ignore_unavailable) | boolean | If an index is closed or missing, an error is returned when ignore_unavailable is set to `false`. Default is set to `false`.
+
+## Examples
+
+Example request and response.
```
Comment (natebower) [inline _opensearch/rest-api/reload-search-analyzer.md:31]: Please make this a complete sentence. Something like "The following are an example request and response" is fine.

File: _opensearch/rest-api/reload-search-analyzer.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+#### Sample request:
+
+````json
+POST /shakespeare/_reload_search_analyzers
+````
+ 
+#### Sample response body:
```
Comment (natebower) [inline _opensearch/rest-api/reload-search-analyzer.md:39]: Remove the colon. I don't normally see "body" included in this heading. Please confirm whether it should just be "Sample response".

**Pull request merged**